up_ex_doc_version.exs

```diff
@@ -0,0 +1,28 @@
+defmodule UpExDocVersion do
+  @mix_exs "mix.exs"
+
+  def up(new_vsn) do
+    content_up =
+      @mix_exs
+      |> File.read!()
+      |> up(new_vsn)
+
+    File.write!(@mix_exs, "#{content_up}\n")
```

Comment:
We probably want to call `deps.get` here or remind the user to.

Reply:
👍 makes sense.

Reply:
15024bd, but it's now a mix task 👍

Reply:
15024bd, but it's now a mix task 👍